routers/web/repo/view.go

```diff
@@ -887,6 +887,26 @@ func renderLanguageStats(ctx *context.Context) {
 	ctx.Data["LanguageStats"] = langs
 }
 
+func renderLatestRelease(ctx *context.Context) {
+	if !ctx.Repo.Repository.UnitEnabled(ctx, unit_model.TypeReleases) || unit_model.TypeReleases.UnitGlobalDisabled() {
```

Comment:
IIRC I do not see other places checking "global disabled".

Only call "UnitEnabled" should be enough?

Reply:
Done.